medicines/web/src/components/mip/index.tsx

```diff
@@ -126,48 +127,65 @@ const Mip: React.FC = () => {
     });
   };
 
+  const loadSearchResults = async (
+    searchTerm: string | string[],
+    page: string | string[],
```

Comment:
They're arrays of strings because they're coming from query string, where anything can be declared numerous times. I can change this up so that we're doing the filtering out before calling this method, but right now the filtering happens inside this.